### ISSUE TITLE
el-get-bundle can not update :after method for existed package

### ISSUE THREAD

**Issue by standin000** [dimitri/el-get#2359]
I have dig the source code, and find el-get-bundle use el-get-do-init which 
call `(el-get-merge-properties-into-status package 'init :noerror t)` and init is not member of required-ops from el-get-diagnosis-properties, so it don't call el-get-save-package-status to update :after method. I can not mitigate it by modify el-get custom variables.

**Comment by npostavs** [comment]
Yes, there is a solution for this in #2264 (but it's not quite ready yet), see also https://github.com/dimitri/el-get/pull/2238#issuecomment-138383276
